README.md

```diff
@@ -73,6 +73,46 @@ We also add following methods to make type-checking more easily:
 - [`find_n`, `first_n`, `last_n`](https://github.com/chanzuckerberg/sorbet-rails#find-first-and-last)
 - [`pluck_to_tstruct`](#pluck_to_tstruct-instead-of-pluck)
 
+#### Enums
+
+If you use [Rails enums](https://guides.rubyonrails.org/active_record_querying.html#enums), `sorbet-rails` will include getters, setters, and scope methods in the rbi file it generates.
+
+Alternatively, you can replace your call to `enum` with `typed_enum`. This will generate [`T::Enum`s](https://sorbet.org/docs/tenum) in your rbi in addition to the standard Rails methods.
```

Comment:
Shouldn't we call out the fact that `sorbet-rails` will generate T::Enum for all enums automatically?

You use `typed_enum` when you want to hide the default getter & setter that Rails provide to require people to use T::Enum.

Another note is that we only hide the signature of the getter from Sorbet. They are still usable at run-time.